oddsgym/envs/base.py

```diff
@@ -193,7 +194,7 @@ def render(self, mode='human'):
         msg : str
             A string with the current balance and the current step.
         """
-        return 'Current balance at step {}: {}'.format(self.current_step, self.balance)
+        return print('Current balance at step {}: {}'.format(self.current_step, self.balance))
```

Comment:
This line is no longer tested in the unit tests (due to the removal of the test).
Also, the ThreeWaySoccer envs also implement a render method (overrides and uses this method). Need to test that this still works.

Reply:
fixed at ThreeWaySoccer. 
There is not test for it, only in the base env, but it was changed to test_info.
should we actually check if it prints to sout?

Reply:
Yeah, the way (that I know) to do it is to "redirect" stdout when testing to a temporary file and then just test that the expected string is in the file. Read pytest documentation on the subject: https://docs.pytest.org/en/stable/capture.html#accessing-captured-output-from-a-test-function